src/PostgREST/Auth.hs

```diff
@@ -74,6 +74,14 @@ addUser identity pass role = do
     [H.stmt|insert into postgrest.auth (id, pass, rolname) values (?, ?, ?)|]
       identity (cs hashed :: Text) role
 
+updateUser :: Text -> Text -> Text -> H.Tx P.Postgres s ()
+updateUser identity pass role = do
+  let Just hashed = unsafePerformIO $ hashPasswordUsingPolicy fastBcryptHashingPolicy (cs pass)
```

Comment:
I've been meaning to get rid of `unsafePerformIO` from the code -- it occurs in another place as well. If you have any ideas here I'd love to hear them.